datastore/tasks/tasks.go

```diff
@@ -0,0 +1,222 @@
+// Copyright 2015 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START all]
+
+// A simple command-line task list manager to demonstrate using the
+// google.golang.org/cloud/datastore package.
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+	"text/tabwriter"
+	"time"
+
+	"golang.org/x/net/context"
+	"google.golang.org/cloud/datastore"
+)
+
+func main() {
+	projID := os.Getenv("DATASTORE_PROJECT_ID")
```

Comment:
Ah yes, I found the wrong one. I see GCLOUD_DATASET_ID used in a couple code samples. They should probably be changed to DATASTORE_PROJECT_ID.